Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -92,6 +93,25 @@ function Write-DTLServiceDeprecationMessageIfRequired
     }
 }
 
+function Publish-Azure-Telemetry
+ {
+   param([object] $deploymentResponse, [string] $jobId)
```

Comment:
for readability keep each parameter in single line.

Reply:
Done